listStyle.css

```diff
@@ -0,0 +1,25 @@
+.chenar{
+    align-content: center;
+    align-self: center;
+   background-color: rgba(0, 169, 169, 0.07);
+    width: 400px;
+    height: 700px;
+    border: 10px solid cadetblue;
+    padding: 50px;
+    margin: auto;
+}
+h1{
+ text-align: center;
+    color: #1d4150;
+    font-family: "Arial Black";
+}
+ul{
+    list-style-position: inside;
+    line-height: 35px;
+    text-size-adjust: auto;
```

Comment:
nu exista asa proprietate in css

Reply:
Am mers prea departe cu experimentele )) Am rezolvat!